.github/workflows/test.yml

```diff
@@ -1,6 +1,12 @@
 name: Run Tests
 
-on: [push]
+on: 
+  push:
+    branches:
+      - master
+  pull_request: 
+    branches:
+      - master
```

Comment:
we can remove this right? What if someone wanted to "stack" prs? We dont really care what the target branch is right?

Reply:
You're right, it wouldn't matter